examples/Test Monitor/NISystemLinkEnterpriseExamples/Results/Results.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using NationalInstruments.SystemLink.Clients.TestMonitor;
```

Comment:
> @Madhan-Reddy-ni We don't quite have the structure right here. We don't want to have an `examples` folder. We just want to have a list of folders at the top-level that each have an example in them. The fix is to rename the `example` folder here to something like `TestMonitor dotnet Example`

@adamarnesen, I have renamed the examples folder name to the DotNet Examples, since we will be adding more DotNet examples related to other services also(not only Test Monitor) and we can add them inside this folder. We will also be adding the Python examples and we can create one more folder as Python Examples and add the python related examples in here.

Reply:
Since the package is available online on GitHub, the user can directly restore the project to get the `TestMonitor` client and run the example.